lib/index.js

```diff
@@ -48,10 +48,14 @@ const express = require('express'),
   compareVersions = require('compare-versions')
 
 function Server (opts) {
+  const bodyParser = require('body-parser')
   const app = express()
   app.use(require('compression')())
   app.use(require('cors')())
-  app.use(require('body-parser').json())
+  app.use(bodyParser.json({limit:'50mb'}))
```

Comment:
50mb is a pretty hefty global default.

How about making this configurable with an environment variable and making the default something more reasonable, like 10mb?

Reply:
Added environment variable and default to 10mb.